src/layouts/SiteLaunchPad/components/SiteLaunchCheckList.tsx

```diff
@@ -44,13 +41,24 @@ export const SiteLaunchChecklistTitle = (): JSX.Element => {
 }
 
 const getTextProps = (index: number, tasksDone: number): TextProps => {
-  const disabled = tasksDone >= index + 1 || tasksDone < index - 1
-  if (disabled) {
+  const tasksPending = tasksDone < index - 1
+  if (tasksPending) {
+    return {
+      color: "interaction.support.disabled-content",
+    }
+  }
+  const taskDone = tasksDone >= index + 1
+
+  if (taskDone) {
     return {
-      color: "interaction.support.disabled",
+      color: "base.content.default",
```

Comment:
i'm just assuming that this is correct because it's quite hard to figure out what exactly `index` refers to here fyi